packages/system/src/constants.ts

```diff
@@ -36,3 +36,29 @@ export const infiniteExtent: CoordinateExtent = [
 ];
 
 export const elementSelectionKeys = ['Enter', ' ', 'Escape'];
+
+export const defaultLabelConfig = {
```

Comment:
I just talked with @peterkogo and he suggested to rename it to "ariaLabelConfig" . We wanted to keep it more flexible in case we would like to add other labels but right now I can think of any other ones. 

Let's rename it and also rename the keys with "...title" to "...ariaRole" (controls.zoomin.title -> controls.zoomin.ariaLabel for example). Let's also rename all keys that start with "a11yDescription" (a11yDescription.node.default -> node.a11Description.default for example). For the users it's easier to understand I think.